workers/src/main/scala/workers/ItcServer.scala

```diff
@@ -57,14 +58,14 @@ object ItcServer extends WorkerServer[IO, ItcMessage.Request] with ItcPicklers {
 
   protected val handler: Logger[IO] ?=> IO[Invocation => IO[Unit]] =
     for {
-      self                               <- IO(dom.DedicatedWorkerGlobalScope.self)
-      cache                              <- Cache.withIDB[IO](self.indexedDB.toOption, "explore-itc")
-      _                                  <- cache.evict(CacheRetention).start
-      matrix                             <- Deferred[IO, SpectroscopyModesMatrix]
-      config                             <- fetchConfig[IO]
-      given TransactionalClient[IO, ITC] <- {
-        given TransactionalBackend[IO] = FetchJSBackend[IO](FetchMethod.GET)
-        TransactionalClient.of[IO, ITC](config.itcURI, "ITC")
+      self                          <- IO(dom.DedicatedWorkerGlobalScope.self)
+      cache                         <- Cache.withIDB[IO](self.indexedDB.toOption, "explore-itc")
+      _                             <- cache.evict(CacheRetention).start
+      matrix                        <- Deferred[IO, SpectroscopyModesMatrix]
+      config                        <- fetchConfig[IO]
+      given FetchClient[IO, ?, ITC] <- {
+        given FetchJSBackend[IO] = FetchJSBackend[IO](FetchMethod.GET)
+        FetchJSClient.of[IO, ITC](config.itcURI.toString, "ITC")
```

Comment:
Why no plain urls in clue clients?

Reply:
`Uri` is an `http4s` class. I removed the `http4s` dependency from the `clue-scalajs` module, since it was used just to import this class. `FetchJSBackend` now exposes what the underlying `fetch` API uses for URLs, which is a plain `String`.

There are `http4s` clients in `clue` which do make use of `Uri`. We should evaluate whether to use these clients.

Finally, I guess we *could* use `java.util.Uri` here, I'm not sure.